pkg/cloud/services/compute/instance.go

```diff
@@ -511,11 +511,15 @@ func (s *Service) getOrCreatePort(eventObject runtime.Object, clusterName string
 		}
 	}
 
+	var portProfile map[string]interface{}
```

Comment:
```suggestion
	portProfile := make(map[string]interface{})
```

Sorry, my fault :)

Reply:
don't be sorry, thanks for your help.

I'll test it locally and send a new patchset!